eos-grub-upgrade

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+# Copyright (C) 2018 Endless Mobile, Inc.
+# Licensed under the GPLv2
+#
+# Extracted from https://phabricator.endlessm.com/w/software/x86-64-boot-paths/
```

Comment:
Maybe "Currently handles the following boot paths (taken from URL):". Also, this has more data than that table, including whether bootia32.efi is present; please could you update the wiki?